config/ceph-config.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
```

Comment:
I'm not discussing if they are useful or not, I strongly believer they are.
However since they are directly connected to the subject of this commit I was thinking that another commit might be more appropriate.
But I'm just being fussy here, so we can merge them with this PR.